packages/flutter/lib/src/material/floating_action_button.dart

```diff
@@ -284,17 +292,28 @@ class FloatingActionButton extends StatelessWidget {
 
     result = RawMaterialButton(
       onPressed: onPressed,
-      elevation: elevation,
-      highlightElevation: highlightElevation,
-      disabledElevation: disabledElevation,
+      elevation: elevation
```

Comment:
I recommend pulling each of these pieces out into a variable like the `foregroundColor`, makes it easier to follow

Reply:
Done